src/demo/DemoTestClientService.java

```diff
@@ -120,6 +122,43 @@ public static void main(String[] args) {
 		}
 	}
 
+	private static String testUpdateClientAssociations(Client client) throws MambuApiException {
+
+		String methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println("\nIn " + methodName);
+		
+		client.setAssignedBranchKey(DemoUtil.getDemoClient().getAssignedBranchKey());
+		client.setAssignedCentreKey(DemoUtil.getDemoClient().getAssignedCentreKey());
+		client.setAssignedUserKey(DemoUtil.getDemoClient().getAssignedUserKey());
+		
+		// null fields not wanted in patch
+		// and keep only the fields defining associations

```

Comment:
There already exists a test for that. The newly added method tests only the newly field supported by the PATCH operation.